draft-song-dns-wireformat-http-01.txt

```diff
@@ -312,37 +315,49 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
    bytes length field in DNS over TCP.
 
    Since this two-byte length field is redundant, when the client proxy
-   receives a DNS message over TCP, it does NOT include the length field
+   receives a DNS message over TCP, it MUST NOT include the length field
    in the message sent to the server.  The length in the content-length
-   header is only the size of the DNS message itself, and does NOT
+   header is only the size of the DNS message itself, and MUST NOT
    include this two-byte length header.
 
-4.  Considerations on DNS over HTTPS
+4.  Security Considerations
 
-   HTTPS is recommended for communication.  It provides privacy and
-   integrity for HTTP sessions.
+   Since this protocol transmits DNS messages, all of the security
+   concerns that stub resolvers and recursive resolvers have with DNS
+   messages apply.  However, since HTTP uses TCP as the underlying
+   protocol, DNS reflection or amplification attacks are not possible.
 
-   As specified in RFC 5246 [RFC5246], both the DNS server and client
-   can be authenticated or not authenticated.  The DNS service providers
-   can decide authenticated pattern on both server and client sides
-   based on their own requirement for security and privacy.  For an open
-   resolver, clients do not require authentication.
+   Since HTTP is used, all of the security concerns of HTTP are also
+   present.
 
 
 
 
+Song, et al.           Expires September 11, 2016               [Page 6]
+
+Internet-Draft          DNS wire-format over HTTP             March 2016
 
 
-Song, et al.            Expires September 1, 2016               [Page 6]
-
-Internet-Draft          DNS wire-format over HTTP          February 2016
+   Servers and clients SHOULD use TLS for communication.  It provides
```

Comment:
it looks you want to express that  "DNS wire-format over HTTP dose not introduce additional security concern, because it is using HTTP or HTTPs to transport DNS massage. ". Can we just say it in that way in the draft?